web/src/main/java/org/cbioportal/web/config/CustomObjectMapper.java

```diff
@@ -43,6 +44,7 @@
     public CustomObjectMapper() {
 
         super.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+        super.enable(SerializationFeature.WRITE_ENUMS_USING_TO_STRING);
```

Comment:
No, Enum names aren't used, it's still the property name like before, this line just changes ObjectMapper to use .toString() method of enum instead of .name() method, because .name() returns PRIMARY_SOLID_TUMOR, but my overriden toString() returns `Primary Solid Tumor`, and we want the second one.